enhancements/machine-config/pin-and-pre-load-images.md

```diff
@@ -126,32 +176,22 @@ configuration parameter to "", but that would affect all images, not just the
 pinned ones. This behavior needs to be changed in CRI-O so that pinned images
 aren't removed, regardless of the value of `version_file_persist`.
 
-The changes to pin the images will be done in a file inside the
-`/etc/crio/crio.conf.d` directory. To avoid potential conflicts with files
-manually created by the administrator the name of this file will be the name of
-the `PinnedImageSet` custom resource concatenated with the UUID assigned by the
-API server. For example, if the custom resource is this:
+The changes to `pinned_images` `CRI-O` configuration will be persisted to a
+static config file `/etc/crio/crio.conf.d/50-pinned-images` directory.
```

Comment:
Double check: `50-pinned-images` is a directory and not a file?

Reply:
file

Reply:
ok, maybe we should remove mention of directory in the end.